crates/bevy_tasks/Cargo.toml

```diff
@@ -20,7 +20,6 @@ futures-lite = "1.4.0"
 event-listener = "2.4.0"
 async-executor = "1.3.0"
 async-channel = "1.4.2"
-instant = { version = "0.1", features = ["wasm-bindgen"] }
```

Comment:
this one is used in a test, maybe it can be moved to a build dependency?

Reply:
Added it as as dev-dependency.